packages/node_modules/nav-frontend-skjema/src/skjema-gruppe.tsx

```diff
@@ -126,13 +136,22 @@ class SkjemaGruppe extends React.Component<SkjemaGruppeProps> {
     /**
      * Feilmelding
      */
-    feil: PT.oneOfType([PT.node, PT.bool])
+    feil: PT.oneOfType([PT.node, PT.bool]),
+    /**
+     * Feilmeldings id
+     */
+    feilmeldingId: PT.string,
+    /**
+     * Option om man sernder feilcontext videre til barnekomponenter
+     */
+    utenFeilContext: PT.bool
```

Comment:
Kan vi kalle denne noe alà `propagateErrorToChildren` eller noe også? "feilContext" hinter til hvor det konkret er implementert i komponenten, men er ikke nødvendigvis lett å forstå for noen som skal bruke den,

Reply:
Godt poeng 👍

Reply:
Foreslår at vi holder props til Norsk intil videre?
Foreslår `propagereErrorTilBarn` som helt norsk
eller `propagereErrorTilChild` som gir mer mening for react.

Reply:
Synes kanskje at default oppførsel her fortsatt bør være at komponenten propagerer dette til children? Da bør vi kanskje kalle denne propen `utenFeilPropagering` eller noe i den duren?

Reply:
Enig. `utenFeilPropagering` bør være deskriptiv nok, siden bruk av prop blir forklart med beskrivelse og eksempel på design.nav.no.